Packs/EDL/Integrations/EDL/README.md

```diff
@@ -46,11 +45,11 @@ Unlike `PAN-OS EDL Management`, this integration hosts the EDL on the Cortex XSO
-| Private Key (Required for HTTPS)   | For use with HTTPS - the private key that the service should use.                                                                                                                                                                                    | False        |
-| Username                           | Uses basic authentication for accessing the list. If empty, no authentication is enforced.                                                                                                                                                           | False        |
-| Password                           | Uses basic authentication for accessing the list. If empty, no authentication is enforced.                                                                                                                                                           | False        |
+| Listen Port                        | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. <br>Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use.  <br> (For Cortex XSOAR 8) Only supported when used on an engine. Otherwise, this setting will be ignored.                                                            | True         |
+| Certificate (Required for HTTPS)   | For use with HTTPS - the certificate that the service should use.  <br> (For Cortex XSOAR 8) Custom certificates are not supported.                                                                                                                                                                                  | False        |
+| Private Key (Required for HTTPS)   | For use with HTTPS - the private key that the service should use.  <br> (For Cortex XSOAR 8) Configuring private API keys is supported only on engines, not on the Cortex XSOAR​​ server.                                                                                                                                                                                  | False        |
+| Username                           | Uses basic authentication for accessing the list. If empty, no authentication is enforced.                                                                                                                                                           | False - For Cortex XSOAR 6.x <br> True - For Cortex XSOAR 8          |
+| Password                           | Uses basic authentication for accessing the list. If empty, no authentication is enforced.                                                                                                                                                           | False - For Cortex XSOAR 6.x <br> True - For Cortex XSOAR 8          |
```

Comment:
```suggestion
| Certificate (Required for HTTPS)   | (For Cortex XSOAR 6.x) For use with HTTPS - the certificate that the service should use.  <br> (For Cortex XSOAR 8 and Cortex XSIAM) Custom certificates are not supported.                                                                                                                                                                                  | False        |
| Private Key (Required for HTTPS)   | For Cortex XSOAR 6.x) For use with HTTPS - the private key that the service should use.  <br> (For Cortex XSOAR 8 and Cortex XSIAM) When using an engine, configure a private API key. Not supported on the Cortex XSOAR​​ or Cortex XSIAM server.                                                                                                                                                                                  | False        |
| Username                           | Uses basic authentication for accessing the list. If empty, no authentication is enforced.                                                                                                                                                           | (For Cortex XSOAR 6.x) False <br> (For Cortex XSOAR 8 and Cortex XSIAM)  Optional for engines, otherwise mandatory.    |
| Password                           | Uses basic authentication for accessing the list. If empty, no authentication is enforced.                                                                                                                                                           | (For Cortex XSOAR 6.x) False <br> (For Cortex XSOAR 8 and Cortex XSIAM)  Optional for engines, otherwise mandatory.     |
```